libs/watt/src/lib/configuration/watt-locale.service.ts

```diff
@@ -14,18 +14,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Injectable } from '@angular/core';
+import { computed, inject, Injectable, LOCALE_ID } from '@angular/core';
 import { DateAdapter } from '@angular/material/core';
 
 import { WattDateAdapter, WattSupportedLocales } from './watt-date-adapter';
-import { Subject } from 'rxjs';
+import { BehaviorSubject } from 'rxjs';
 import { dayjs } from '@energinet-datahub/watt/date';
+import { toSignal } from '@angular/core/rxjs-interop';
 
 @Injectable({
   providedIn: 'root',
 })
 export class WattLocaleService {
-  onLocaleChange$ = new Subject<WattSupportedLocales>();
+  private locale: WattSupportedLocales = inject(LOCALE_ID) as WattSupportedLocales;
+
+  onLocaleChange$ = new BehaviorSubject(this.locale);
+  current = toSignal(this.onLocaleChange$, { requireSync: true });
```

Comment:
suggestion: thinking about it, isn't this just a fancy `signal` with initial value? I wonder whether simply 👇 will do it 🤔

```ts
locale = signal(inject<WattSupportedLocales>(LOCALE_ID));
```

Reply:
Good question, I would assume that works, although it looks a bit funky 😄 I'm not sure how you want me to use that signal though, can you show me the rest of the implementation?

Reply:
Something like this

```ts
current = signal(inject<WattSupportedLocales>(LOCALE_ID));

isDanish = computed(() => this.current() == 'da');
isEnglish = computed(() => this.current() == 'en');

constructor(private dateAdapter: DateAdapter<unknown>) {}

async setActiveLocale(locale: WattSupportedLocales): Promise<void> {
  if (locale === 'da') {
    await import('dayjs/locale/da');
  }

  if (locale === 'en') {
    await import('dayjs/locale/en');
  }

  dayjs.locale(locale);
  (this.dateAdapter as WattDateAdapter).setActiveLocale(locale);
  this.current.set(locale);
}
```

Can see that `watt-datepicker.component.ts` will need to be adjusted slightly should we go with this.

Reply:
Good idea and it appears to work 🎉 

https://github.com/Energinet-DataHub/greenforce-frontend/pull/3761/commits/acab1188056030b06eed71590587a4655a828a16

Reply:
Thinking about it, let's not introduce more changes just for the sake of it. We can make it in a follow-up PR if we still think this is a good idea.